rainbowhat/src/main/java/com/google/android/things/contrib/driver/rainbowhat/RainbowHat.java

```diff
@@ -92,4 +92,8 @@ public static Gpio openLed(@LedPin String pin) throws IOException {
     public static Apa102 openLedStrip() throws IOException {
         return new Apa102(BUS_LEDSTRIP, Apa102.Mode.BGR);
     }
+
+    public static Apa102 openLedStripReversed() throws IOException {
```

Comment:
```java
public static Apa102 openLedStrip(Apa102.Direction direction) {... }
``` 
?

Given there are only two possible `Direction`s I think an explicit name would be better but can implement your suggestion just the same.